english/rib-eye/rib-eye.md

```diff
@@ -0,0 +1,13 @@
+All this talk of chocolate and donuts. You want a steak. A thick, juicy, salty steak. But, what kind? How about a Rib Eye? But where dost thou find a slab of beautiful bovine around here? 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.